x-pack/plugins/observability_onboarding/server/index.ts

```diff
@@ -16,6 +16,7 @@ const configSchema = schema.object({
   ui: schema.object({
     enabled: schema.boolean({ defaultValue: true }),
   }),
+  elasticAgentVersion: schema.string({ defaultValue: '8.8.0' }),
```

Comment:
i didn't find a way to do this, other than calling the github API directly (https://api.github.com/repos/elastic/elastic-agent/releases/latest). I'm not sure we want to add a dependency on Github here, since we host our own build artifacts (they are not primarily distributed via github). Also, the github API does not return direct download links, only version number, so we still need to build download URLs and hope that the pattern stays consistent and the file exists. Ideally, i would love to have Kibana be able to have direct download links available to for each platform, but it doesn't seem like it exists.

Reply:
Will `GET /api/status` help us in this case? AFAIU stack should be aligned and that API return a `version.number` property that might help us here

Reply:
i'm not sure we can depend on the using the kibana version to construct the download URLs. For example, on `main` the kibana version is `8.9.0` and there are often pre-release and snapshot builds that would result in the Elastic Agent download failing. The gitub release API is much more reliable, but i'll investigate if there is a better way to obtain the latest version. We could also hard code a version for now and add a look up later after some discovery.

Reply:
For this PR, i removed the config and set the value in a hardcoded constant. we can address the correct way to populate the Elastic Agent version / download URLs in a follow-up issue.